lib/resources/aws/aws_s3_bucket.rb

```diff
@@ -35,6 +35,11 @@ def public?
       bucket_policy.any? { |s| s.effect == 'Allow' && s.principal == '*' }
   end
 
+  def has_default_encryption_enabled?
+    return unless @exists
```

Comment:
[nitpick] Predicate methods should only ever return explicit `true` or `false` values. So `return false unless @exists` would be the more accurate guard statement.